packages/nc-gui/components/smartsheet/expanded-form/index.vue

```diff
@@ -324,15 +324,9 @@ const onDeleteRowClick = () => {
 const onConfirmDeleteRowClick = async () => {
   showDeleteRowModal.value = false
   await deleteRowById(primaryKey.value)
-  message.success('Row deleted')
```

Comment:
double check if this `message.success` is no longer required so deleted by mistake.